features/steps/core.py

```diff
@@ -66,6 +67,12 @@ def set_config(context, config_file):
             cf.write("version: {}".format(__version__))
 
 
+@when('we open the editor and exit')
+def open_editor_and_exit_without_entering_text(context):
+    with patch('jrnl.util.get_text_from_editor', return_value=""):
```

Comment:
Patching the subprocess.call is a bit cleaner because you would still get the proper console output. Alternatively you can create a function that prints the expected output and add it to `patch(side_effect=<print_func>)`.

Reply:
> Alternatively you can create a function that prints the expected output and add it to patch(side_effect=<print_func>)

Seems messy to me, and would require changing both the test and the code if the message was ever modified. I'd rather patch `subprocess.call`